app/models/nearby_facility.rb

```diff
@@ -48,6 +59,32 @@ def make_linestring(polygon)
       polygon.map { |point| "#{point[1]} #{point[0]}" }.join(',')
     end
 
+    def parse_isochrone(response_json)
+      response_json.try(:[], 'resourceSets')
```

Comment:
Is it possible to write this  `try` chain with safe navigation operators (`&.`)? Not asking you to change it (though it makes me slightly sad that it takes up this many lines); only asking because I tried a couple permutations but didn't get the syntax right.

Reply:
Possibly. There is some weirdness with the safe navigation operator if something in the chain returns true/false, but I will test it